src/platform/platform.kernel.vcxproj

```diff
@@ -104,4 +110,4 @@ mc.exe -um -h $(SolutionDir)build\winkernel\$(Platform)_$(Configuration)_schanne
   </ItemDefinitionGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```

Comment:
VS ate the newline at the end of the file again

Reply:
Uggggg! I will fix this.